lib/ansiblelint/rules/ComparisonToLiteralBoolRule.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2016, Will Thames and contributors
+# Copyright (c) 2018, Ansible Project
+
+from ansiblelint import AnsibleLintRule
+import re
+
+
+class ComparisonToLiteralBoolRule(AnsibleLintRule):
+    id = '601'
+    shortdesc = "Don't compare to literal True/False"
```

Comment:
there are use cases where comparing to `False` is required if the var is `null`.

Reply:
@privateip I guess `var != False` is more elegant than `var is not None and not var` or `not var|default(True)`

Perhaps it should just be ComparisonToTrue

Reply:
@willthames `!=` comparisons with bools are highly discouraged in Python. So to Pythonistas it looks just as ugly as Java :D

Reply:
Also, identify comparison is not exactly the same as comparison by value.